utils/prepare_config.py

```diff
@@ -58,6 +65,8 @@ def configure_server(app: str, global_rounds: int, trusts: List[str], ignore_res
         server_config["min_clients"] = len(trusts)
         server_config["workflows"][2]["args"]["participating_clients"] = trusts
         server_config["workflows"][1]["args"]["ignore_result_error"] = ignore_result_error
+        server_config["components"][2]["path"] = f'nvflare.app_common.aggregators.intime_accumulate_model_aggregator.{aggregator}'
```

Comment:
I've just realised for this to work best, I think we're gonna have to make sure we specify the nvflare component name instead.
```suggestion
        server_config["components"][2]["name"] = aggregator
```
You'll also need to remove the "path" property from the component config in the base app as well https://github.com/AI4VBH/FLIP-fl-base-application/blob/develop/src/config/config_fed_server.json

NVFlare can use either the "name" or "path" property to reference components but "name" is only for stuff built into nvflare itself